client/src/components/LeafletOsmMap/SavedPlaces.js

```diff
@@ -0,0 +1,159 @@
+import {h, Component} from "preact";
+import {makeRequest} from "../../js/server-requests-utils";
+
+/**
+ * Leaflet related imports: leaflet, pouchdb module, and routing machine module
+ */
+import '../../../node_modules/leaflet/dist/leaflet.css';
+import '../../../node_modules/leaflet-routing-machine/dist/leaflet-routing-machine.css';
+import L from '../../js/leaflet-tileLayer-pouchdb-cached';
+
+/* Set all settings for saved place icon
+Waiting for new icon for favorites
+Full list of options: http://leafletjs.com/reference-1.3.0.html#icon
+ */
+const FAV_MARKER_ICON = L.icon({
+  iconUrl: '../../assets/icons/leaflet/marker-icon-fav-2x.png',
+  className: 'favorites',
+  iconSize: [25, 41], //native aspect ratio: [28, 41]
+});
+
+
+const CURRENT_USER = makeRequest('GET', 'user')
```

Comment:
You don't need to get user credentials. There is control on the backend that verifies that user is authenticated prior to giving access to saved data in the database. This happens for all access to saved data - see `~/routes/search.js`: `verifyToken` is called on all those end points. Once user is logged in with non-expired token, user will automatically have access to their respective saved data.

Reply:
Thanks @jkwening! Honestly, I didn't realize that the token was the user_id and that the backend was adding it to the request as the id.  I'll modify the file accordingly

Reply:
This is now corrected.